cpu/esp32/Makefile.features

```diff
@@ -1,5 +1,8 @@
 # Features that are provided by the CPU independent on the board
+FEATURES_PROVIDED += arch_32bit
+FEATURES_PROVIDED += arch_esp32
 FEATURES_PROVIDED += cpp
+# Features that are provided by CPU independent on the board
```

Comment:
I think this comment could be removed. It's almost the same as the one line 1.

Reply:
maybe, but that would be an unrelated change.

Reply:
This is an addition from this PR, so not something that is changed. Please remove.

Reply:
removed